package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaxios",
-  "version": "0.0.0",
+  "version": "2.0.2",
```

Comment:
odd that this was `0.0.0`?

Reply:
🤣 yeah.  It's because I was previously using `semantic-release`, which sets the version at deploy time based on the previous published version.  Good news is it looks like this is the right version!